fe/fe-core/src/main/java/com/starrocks/catalog/ScalarType.java

```diff
@@ -349,6 +353,10 @@ public static ScalarType createHllType() {
         return type;
     }
 
+    public static ScalarType createJsonType() {
+        return new ScalarType(PrimitiveType.JSON);
```

Comment:
Is Json a scalar type?

Reply:
Yes, the differences between scalar and compound type is that the laster is consisting of the former. 
JSON is not constructed from other scalar types, so itself is a scalar type.